src/routes/resources/+page.svelte

```diff
@@ -1,96 +1,51 @@
 <script lang="ts">
-	const booksFromPublisher = [
-		{
-			name: 'Svelte 3 Up and Running',
-			link: 'https://www.amazon.com/dp/B08D6T6BKS/',
-			author: 'Alessandro Segala'
-		},
-		{
-			name: 'Svelte and Sapper in Action',
-			link: 'https://www.manning.com/books/svelte-and-sapper-in-action',
-			author: 'R. Mark Volkmann'
-		},
-		{
-			name: 'SvelteKit Up and Running',
-			link: 'https://sveltekitbook.dev',
-			author: 'Dylan Hildenbrand'
-		},
-		{
-			name: 'Svelte with Test-Driven Development',
-			link: 'https://www.amazon.com/dp/1837638330',
-			author: 'Daniel Irvine'
-		}
-	];
-	const booksSelfPublished = [
-		{
-			name: 'Svelte Handbook',
-			link: 'https://flaviocopes.com/page/svelte-handbook/',
-			author: 'Flavio Copes'
-		},
-		{
-			name: 'Simple Svelte',
-			link: 'https://wfq.gumroad.com/l/simple_svelte',
-			author: 'Darren Wang'
-		}
-	];
+	import books from './books.json';
+	import extensions from './extensions.json';
+	import misc from './misc.json';
+	import videos from './videos.json';
 </script>
 
 <div>
 	<h2>Books</h2>
 
-	<div>There are a few books from major publishers:</div>
 	<ul>
-		{#each booksFromPublisher as { name, link, author }}
-			<li><a href={link} target="_blank">{name}</a> by {author}</li>
-		{/each}
-	</ul>
-
-	<div>As well as a couple self-published books:</div>
```

Comment:
We lose the indication of self-published, which is a different involvement and process

Reply:
Good point, fixed!